erts/emulator/beam/jit/arm/instr_bs.cpp

```diff
@@ -1443,8 +1482,174 @@ struct BscSegment {
 
     Uint error_info;
     Sint effectiveSize;
+    enum action { NONE, ACCUMULATE_FIRST, ACCUMULATE, STORE } action;
```

Comment:
Using `enum class` will prevent setting this to invalid values by accident, e.g. `seg.action = 0;` will no longer work.
```suggestion
    enum class action { NONE, ACCUMULATE_FIRST, ACCUMULATE, STORE } action;
```